src/app/main.cpp

```diff
@@ -187,6 +189,19 @@ int main(int argc, char *argv[])
         std::cerr << "Couldn't set environment variable...\n";
 
 #ifndef DISABLE_GUI
+
+    // Set dark theme
+    if (params.darkTheme) {
+        QFile file(":/qdarkstylesheet.qss");
```

Comment:
I don't know :) I'm into customisations and support the idea that applications styles have to be controllable by a user. @sledgehammer999 ? Could you comment on this, please?